curriculum/challenges/_meta/learn-intermediate-css-by-building-a-cat-painting/meta.json

```
@@ -0,0 +1,357 @@
+{
+  "name": "Learn Intermediate CSS by Building a Cat Painting",
+  "isUpcomingChange": true,
+  "usesMultifileEditor": true,
+  "hasEditableBoundaries": true,
+  "dashedName": "learn-intermediate-css-by-building-a-cat-painting",
+  "order": 20,
```

According to the spreadsheet, this project is listed right after the picasso painting. So that would mean the order is supposed to be 12 and not 20. 

But my understanding was this project is meant to replace the picasso paining. 
Is that right?

If so, then I guess the number should be 11. 

I would reach out to the team about that.

---

I'll audit the order once all of the projects are in.

---

@naomi-lgbt and @jdwilkin4, can I go ahead and resolve this conversation?

---

I was incorrect - this is the RWD not the JS, which means it's already on production. What we'll need to do is use the correct order in this PR:

```suggestion
  "order": 11,
```

Then once this is merged, merge #50768 before deploying.